backend/src/scraper/bestbuy_scraper.py

```diff
@@ -129,20 +114,16 @@ def _product_reviews(product_name: str) -> Dict:
                 if similarity >= 0.65:
                     compatible_sku = product.get('sku')
                     try:
-                        search_sku = requests.get(f"https://www.bestbuy.ca/api/v2/json/reviews/{compatible_sku}?page=1&source=us", headers = params.get('headers'))
```

Comment:
Can you clarify why we're increasing the page size? Does this mean that we're pulling more reviews?

Reply:
Same amount of reviews as before but only one API call instead of multiple. Say a product has 100 reviews, before we were doing 10 API calls of 10 reviews per call, and now we're doing 1 API call of 100 reviews.